Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3-onbuild
+
+WORKDIR .
+COPY requirements-test.txt .
+COPY setup.py .
```

Comment:
are you sure these two lines (starting with COPY) are needed? Try building the image without them and see if it works anyway, if it does remove them

Reply:
no, it doesnt work if without COPY

Reply:
you can surely remove `requirements-test.txt .` as we don't need to install those here (those are needed to run the automated test suite)

Reply:
is the COPY setup.py . should be remove to?

Reply:
not if it doesn't work without it ;-)

Reply:
okay, done